app/policies/diary/wizard_policy.rb

```diff
@@ -0,0 +1,15 @@
+module Diary
+  class WizardPolicy < ApplicationPolicy
+    def show?
+      @auth_context.role_model.is_a?(FosterParent)
```

Comment:
We definitely have to authorise the `FosterParent` to be able to only create a `DiaryEntry` for their own `Child`. Now I'm not sure if this should be done by this `WizardPolicy` or ideally the `DiaryEntryPolicy` itself 🤔 

This applies to the `update?` action as well, primarily there actually.